tests/integration/list/resources/resources_integ_base.py

```diff
@@ -0,0 +1,32 @@
+import os
+from unittest import TestCase
+from pathlib import Path
+import uuid
+import shutil
+import tempfile
+from tests.integration.list.list_integ_base import ListIntegBase
+
+
+class ResourcesIntegBase(ListIntegBase, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def setUp(self):
+        super().setUp()
+
+    def tearDown(self):
+        super().tearDown()
```

Comment:
AFAIK, you don't need to define methods if you are just going to call `super`.

Reply:
Thanks, I've removed those